testsV2/postgres_test.go

```diff
@@ -86,3 +90,49 @@ func TestPostgresAuthWithCredentialsFile(t *testing.T) {
 		[]string{"--credentials-file", path, *postgresConnName},
 		"postgres", dsn)
 }
+
+func TestAuthWithGcloudAuth(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping Postgres integration tests")
+	}
+	requirePostgresVars(t)
+
+	// The following configures gcloud using only GOOGLE_APPLICATION_CREDENTIALS.
+	configureGcloud := func(t *testing.T) func() {
+		dir, err := ioutil.TempDir("", "cloudsdk*")
+		if err != nil {
+			t.Fatalf("failed to create temp dir: %v", err)
+		}
+		os.Setenv("CLOUDSDK_CONFIG", dir)
```

Comment:
Can we add a comment for why we need to do this?

Reply:
Done. In short, it's to provide a temporary directory to configure gcloud without messing with any existing config. I've added that to the comment.